test_app/scripts/container_startup_uwsgi.sh

```diff
@@ -12,6 +12,7 @@ echo "settings.DATABASE ..."
 $PYTHON manage.py shell -c 'from django.conf import settings; print(settings.DATABASES)'
 
 $PYTHON manage.py migrate
+$PYTHON manage.py collectstatic --clear --noinput
```

Comment:
The `--clear` option would blow away the existing file in the test_app/static/ dir when I had it there, but using the new folder resolves this, so that'll only contain collected static files.